CONTRIBUTING.md

```diff
@@ -45,18 +54,7 @@ mkdir -p ~/.hah/data
 mongod --dbpath ~/.hah/data
 ````
 
-## Start NPM
+## Start Server
 ````
 npm start
 ````
-
-
-## Install mongoose 
-````
-npm install mongoose
-````
-
-## Install bluebird
-````
-npm install bluebird
```

Comment:
Because the `mongoose` and `bluebird` packages are listed in `package.json`, they will be installed by running either `npm install` or now `yarn` (both do the same thing)